### PR TITLE
Enabled ItElasticLoggingFluentd to run on OKD

### PR DESCRIPTION
**Pull request by hzhao-github**
Jenkins

https://build.weblogick8s.org:8443/job/wko-okd-test/100
https://build.weblogick8s.org:8443/job/wko-okd-test/101
https://build.weblogick8s.org:8443/job/wko-okd-test/102
https://build.weblogick8s.org:8443/job/wko-okd-test/103

### CODE REVIEW

**Review by maggiehe00 — APPROVED** [review]
LGTM

**Comment by robertpatrick** [comment]
Please update the copyright to 2022.

**Review by anpanigr — COMMENTED** [review]
Change the copyright to reflect year 2022

**Comment by hzhao-github** [comment]
@robertpatrick @anpanigr done. Thanks

File: integration-tests/pom.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!-- Copyright (c) 2020, 2021, Oracle and/or its affiliates.
+<!-- Copyright (c) 2020, 2021, 2022, Oracle and/or its affiliates.
```
Comment (anpanigr) [inline integration-tests/pom.xml:1]: It should be 2020, 2022,

Comment (hzhao-github) [reply integration-tests/pom.xml:1]: ? why no 2021?

Comment (anpanigr) [reply integration-tests/pom.xml:1]: Can you please follow the Corporate instruction  as posted by Ryan  in slack channel 
https://proddev-paas-fmw.slack.com/archives/C998UPSCT/p1641489147003000

Comment (hzhao-github) [reply integration-tests/pom.xml]: Changed

**Review by anpanigr — APPROVED** [review]
lgtm

**Pull request merged**